packages/suite-data/files/translations/cs.json

```diff
@@ -751,22 +755,26 @@
   "TR_FINAL_HEADING": "Nastavení je dokončeno!",
   "TR_FIND_OUT_MORE_INFO": "Zjistit více informací",
   "TR_FIRMWARE": "Firmware",
+  "TR_FIRMWARE_HASH_MISMATCH": "Váš Trezor používá neoficiální firmware. Okamžitě kontaktujte help@trezor.io.",
   "TR_FIRMWARE_INSTALLED": "Perfektní. Je nainstalován nejnovější firmware. Je čas pokračovat\n",
-  "TR_FIRMWARE_INSTALLED_TEXT": "Toto zařízení má již nainstalovanou verzi firmwaru: {version}",
+  "TR_FIRMWARE_INSTALLED_TEXT": "V tomto zařízení je nainstalován firmware {version} ({type}).",
   "TR_FIRMWARE_IS_POTENTIALLY_RISKY": "Aktualizace firmwaru je potenciálně riziková operace. Pokud se něco pokazí (přerušení kabelu apod.), zařízení může skončit ve vymazaném stavu, což fakticky znamená ztrátu všech mincí.",
   "TR_FIRMWARE_IS_UP_TO_DATE": "Firmware je připraven",
   "TR_FIRMWARE_NEW_FW_DESCRIPTION": "Je dostupná nová verze firmwaru. Nyní můžete buď aktualizovat zařízení nebo pokračovat a aktualizovat později.",
   "TR_FIRMWARE_PARTIALLY_UPDATED": "Firmware byl částečně aktualizován",
   "TR_FIRMWARE_REINSTALL_FW_DESCRIPTION": "Vaše zařízení je již aktualizováno na nejnovější firmware. V případě potřeby můžete firmware přeinstalovat.",
   "TR_FIRMWARE_STATUS_INSTALLATION_COMPLETED": "Dokončeno",
-  "TR_FIRMWARE_SUBHEADING": "Trezor je dodáván bez nainstalovaného firmwaru, abyste mohli ihned začít používat nejnovější funkce. Pravost nainstalovaného firmwaru je vždy kontrolována při spuštění zařízení. Pokud firmware není správně podepsán společností SatoshiLabs, zobrazí se na zařízení Trezor varování.",
+  "TR_FIRMWARE_SUBHEADING_BITCOIN": "Lehký firmware podporující pouze operace s Bitcoinem.",
+  "TR_FIRMWARE_SUBHEADING_NONE": "Trezor je dodáván bez firmwaru. Abyste mohli zařízení bezpečně používat, je třeba do něj nahrát nejnovější firmware. Pokud jste super fanouškem BTC, můžete si nainstalovat <button>firmware určený pouze pro Bitcoin</button>.",
```

Comment:
"Pokud jste super fanouškem BTC" 😄 to mi v češtině nějak zní divněji než v angličtině... @hynek-jina